### PR TITLE
Benaychh

### PR DESCRIPTION
**Pull request by BenAychh**
Hey guys, help me find the mistakes I made in here. Thanks :)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -244,6 +250,9 @@ update-build: upload-build
 			"ParameterKey=ContainerPort,ParameterValue=${CONTAINER_PORT}" \
 			"ParameterKey=ListenerRulePriority,ParameterValue=${LISTENER_RULE_PRIORITY}" \
 			"ParameterKey=HealthCheckPath,ParameterValue=${HEALTH_CHECK_PATH}" \
+			"ParameterKey=SlackWebhook,ParameterValue=${SLACK_WEBHOOK}" \
```
Comment (BenAychh) [inline Makefile:253]: Not sure this actually does anything. From what I have read, lambda's pretty much have to be updated manually? I know for sure I can't get the code to update.

Comment (mathomas) [reply Makefile:253]: So changing any of these parameters doesn't cause the Lambda's environment to be updated?

Comment (mathomas) [reply Makefile:253]: Maybe demonstrate this to me.  I have no depth in doing Lambdas via Cfn, but it seems wrong that you can't have them update via Cfn updates.  Maybe it's dependent on the Lambda versioning stuff in some way (?).

**Review by mathomas — APPROVED** [review]
Nice work!

**Pull request merged**